package.json

```diff
@@ -41,7 +46,8 @@
     },
     "scripts": {
         "test": "nyc mocha tests/rules/",
-        "lint": "eslint ."
+        "lint": "eslint .",
+        "build": "webpack --output-path=build"
```

Comment:
May be good to clear the bulid dir as part of the build script, to make sure that if the file exists than it is for sure a fresh built one